css/custom.css

```diff
@@ -646,9 +646,9 @@ a .abbr[data-original-title], a abbr[title]{
 @media (min-width: 768px) {
     .hardwareAcceleration .col-lg-2dot5 {
         -webkit-box-flex: 0;
-        -ms-flex: 0 0 20%;
-        flex: 0 0 20%;
-        max-width: 20%;
+        -ms-flex: 0 0 25%;
```

Comment:
Instead of adjusting the % for .col-lg-2dot5, you should probably just update the class on the main page matrix to col-lg-2dot4 for each of the EP selections